05-classification.Rmd

```diff
@@ -477,17 +499,17 @@ cancer %>%
 
 *Click and drag the plot above to rotate it, and scroll to zoom. Note that in
 general we recommend against using 3D visualizations; here we show the data in
-3D only to illustrate what "higher dimensions" look like for learning
-purposes.*
+3D only to illustrate what "higher dimensions" and "nearest neighbours" look like,
+for learning purposes.*
 
 **Summary**
 
 In order to classify a new observation using a K-nearest neighbour classifier, we have to:
 
-1. Compute the distance between the new observation and each observation in the training set
-2. Sort the data table in ascending order according to the distances
-3. Choose the top $K$ rows of the sorted table
-4. Classify the new observation based on a majority vote of the neighbour classes
+1. Compute the distance between the new observation and each observation in the training set.
+2. Sort the data table in ascending order according to the distances.
+3. Choose the top $K$ rows of the sorted table.
+4. Classify the new observation based on a majority vote of the neighbour classes.
 
 
 ## K-nearest neighbours with `tidymodels`
```

Comment:
The first sentence below is awkward, let's change it to something like:

"Coding the K-nearest neighbour algorithm in R ourselves would get complicated 
if we wanted to predict the label/class for multiple new observations, 
or if our data had more than two variables."

Reply:
Also below, let's call `tidymodels` a meta package, as we refer to the `tidyverse` package.

Reply:
done and opened issue on global formatting pass on what we call packages